modules/core/05-port/types/router_v2.go

```diff
@@ -5,67 +5,109 @@ package types
 import (
 	"errors"
 	"fmt"
-
-	"golang.org/x/exp/maps"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// AppRouter contains a map from module name to an ordered list of IBCModules
-// which contains all the module-defined callbacks required by ICS-26.
+const sentinelMultiPacketData = "MultiPacketData"
```

Comment:
The spec will decide the mechanism for determining the commitment structure, so this may change. Maybe we should add a comment so readers don't jump to conclusions